hack/rpm-deps.sh

```diff
@@ -86,7 +86,7 @@ launcherbase_base="
 "
```

Comment:
I know that virtiofsd is not officially supported (yet), but the current launcher image includes virtiofsd (it's part of qemu-kvm-core). However, in CentOS 9, virtiofsd is packaged independently.

Since the C version is now deprecated in favor of the Rust version, could the virtiofsd-1.4.0 package be included? (for both aarch64 and x86_64).

Reply:
@germag good catch, I'll take care of that :)

Reply:
Done!